tests/JMS/Serializer/Tests/Serializer/ArrayTest.php

```diff
@@ -0,0 +1,1307 @@
+<?php
```

Comment:
Let's not replicate the entire test suite. One simple test per method should be ok.